dev/user.clj

```diff
@@ -20,7 +20,7 @@
 
   (let [new-system (resolve 'kixi.heimdall.system/system)]
     (alter-var-root #'kixi.heimdall.application/system
-                    (constantly (new-system)))))
+                    (constantly (new-system :development)))))
```

Comment:
this is for use in the repl.  In prod we pass in a parameter to the jar for the profile (see bootstrap.clj)